Fundamentals/Data/Relations/index.md

```diff
@@ -0,0 +1,103 @@
+
+Umbraco Relations are 2-way, when you create a relation between two different types of entities, you can alway find one entity from the other and vice versa. eg you could list out in the backoffice all the pages that a content banner had been picked on.
+
+## Relation Types
+
+In order to create and use Relations in your Umbraco Website, you need to define a 'Relation Type' to specify what the two types of entities will be relatable, and to give the Relation Type an alias, so you can test if two items are related for a particular 'relation type'. Two items might be related under multiple different Relation Types, and you might be only interested in your 'Related Language Page' Relation Type.
+
+### Create a Relation Type
+
+Visit the Settings section of the Umbraco backoffice and you'll find the 'Relation Type' tree
+
+![Relation Types Tree](images/relation-types-tree.png)
+
+If you expand the Relation Type tree, you can see that Umbraco 'ships' with some default Relation Types, that helps the backoffice function:
+
+![Default Relation Types](images/default-relation-types.png)
+
+for example there is a Relation Type that tracks when Media is picked in Content to be able to provide the functionality of warning an editor if they try to delete a Media Item that it is 'in use'. There is a Relation Type, to help 'restore' deleted content back to the place it was deleted from in the Recycle Bin.
+
+Right click the 'Relation Types' folder to create your new Relation Type
+
+![Create Relation Type](images/create-relation-type.png)
+
+Provide the Relation Type with a name (this will generate it's alias).
+
+Choose the 'direction' of the relationship, eg usually Bidirectional to get the benefits of relations.
+
+Define the type of one object in the relation (confusingly this is called 'Parent' but that only defines which column in the database this value is stored in, for a bidirectional relationship it doesn't matter which type of entity is defined as the parent or the child, but if there is a natural 'one thing' will be related to lots of 'other things', then choose that thing as the parent. (really they should be Uncles/Aunts and Nephews/Nieces - but there isn't a common non-gendered term for this family relationship, although Piblings and Niblings has started to become popular).
+
+Choose the different types for each entity in the relationship.
+
+Currently the available types are:
+
+Document(Content), Media, Member, Document Type, Media Type, Member Type, Member Group, Data Type, Root, Recycle Bin
+
+eg. for relating Members to their uploaded Images, we might create a 'Member Images' relation
+
+![Member Images](images/member-images.png)
+
+'Is Dependency' - in the functionality to prevent an item from being deleted if it's in use, some performance gains have been found when querying if a media item is a dependency - by adding an 'Is Dependency' field to the relationship, generally for your custom relations, you'll set this to false.
```

Comment:
I think actually this could just be:

Select **Is Dependency**  as **Yes**  if you wish this Relation Type to be used in Umbraco's core tracking functionality; that warns editors when trying to delete entities if they are 'used' in a Relation Type marked as 'Is Dependency'

Reply:
@eshanrnh tweaked and pushed!